core/modules/filters/x-listops.js

```diff
@@ -23,13 +23,22 @@ Extended filter operators to manipulate the current list.
         return results;
     };
 
+    /*
+    Return the number the str represents
+    Return default if it's not a number
+    */
+    function getInt( str, deflt ) {
```

Comment:
House style is not to have spaces in parameter lists: `function getInt(str,deflt) {`